### DISCUSSION TITLE
codigo

### DISCUSSION THREAD

**Discussion by gjmosquerac** [community/community#73429] [New to GitHub]
Dame la estructura del código en HTML, css y PHP por separados y carpeta archivos de IMG, pagima2.php-includes index.php para crear un programa que calcule la tarifa en dólar de un viaje en taxi, que dependa de la hora del día y la distancia recorrida, siguiendo las características o especificaciones: 
* Aplicar contadores, acumuladores, rutas relativa y absoluta.
*Aplicar las estructuras de control selectivas que sea necesario o recomendable (if, elseif, else, y ver cómo se aplica usando el switch).
* ubicar la carpeta htdocs: Por defecto, la ubicación de esta carpeta en Windows es C:\xampp\htdocs, mientras que en macOS y Linux es /opt/lampp/htdocs.
* Cada vez que cambien algo en el código deben recargar la pagina.

**Comment by queenofcorgis** [comment]
Hi @gjmosquerac , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
